src/Markdig/Renderers/Normalize/Inlines/LinkInlineRenderer.cs

```diff
@@ -44,7 +44,7 @@ protected override void Write(NormalizeRenderer renderer, LinkInline link)
                 {
                     renderer.Write('(').Write(link.Url);
 
-                    if (!string.IsNullOrEmpty(link.Title))
+                    if (link.Title is { Length: > 0 })
```

Comment:
We don't *have* to change everything to pattern matching. Last time I checked it still had worse code generation, so as long as we don't do it on hot paths is should be fine.

Reply:
Agreed -- and have seen similar reports of worse codegen for this pattern. This is another one-off to ensure for the analyzer to flow through the non-null check.